data/eos-updater-flatpak-installer.service.in

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=Endless OS Automatic Updater - Post Boot Flatpak Installer
+Wants=local-fs.target
+After=local-fs.target
+ConditionKernelCommandLine=!eos-updater-disable
+
+# Only on updates
+Before=multi-user.target systemd-update-done.service
+ConditionNeedsUpdate=/var
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+ExecStart=@libexecdir@/eos-updater-flatpak-installer
+
+# Sandboxing
+# FIXME: Enable more of these options once we have systemd > 229
```

Comment:
`FIXME` can be dropped, because we have systemd > 229, and you _have_ enabled more of these options.